.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+python:
+  - "2.7"
```

Comment:
I get an error when trying to install for python 2.7 for some reason.  Just remove this line.